legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/testable/service/assertion/ServiceTestAssertionEvaluator.java

```diff
@@ -66,6 +66,10 @@ else if (testAssertion instanceof EqualToJson)
             {
                 actualJson = (String) ((ConstantResult) result).getValue();
             }
+            else if (result instanceof RelationalResult)
```

Comment:
I think store specific logic should not come to this module & do we need this, should we just default to streaming result with default serialization format
and have an option in test specification to specify. What are your thoughts